GPU/GPUTracking/Definitions/GPUDefGPUParameters.h

```diff
@@ -71,6 +71,9 @@
   #define GPUCA_LB_GPUTPCGMMergerFinalize_2 256
   #define GPUCA_LB_GPUTPCCompressionKernels_step0attached 64, 2
   #define GPUCA_LB_GPUTPCCompressionKernels_step1unattached 512, 2
+  #define GPUCA_LB_GPUTPCDecompressionKernels_step0attached 64, 2
+  #define GPUCA_LB_GPUTPCDecompressionKernels_step1unattached 512, 2
+  #define GPUCA_LB_GPUTPCDecompressionUtilKernels_sortPerSectorRow 512, 2
```

Comment:
For reference: Did you actually tune these values, also the ones below, for the respective GPU models?
Or did you just copy them?
If they are not tuned, we should remove them here, and just leave 
```
#ifndef GPUCA_LB_GPUTPCDecompressionKernels_step0attached
    #define GPUCA_LB_GPUTPCDecompressionKernels_step0attached 256
  #endif
```
etc below.

Reply:
I did not tune them, I just copied the values from the compression kernels. I will remove them and leave just the general ones below.